lib/src/tracker.ts

```diff
@@ -97,4 +97,16 @@ export class Tracker extends RelewiseClient {
             }
         });
     }
+
+    public async trackSearchTerm({ term, languageName, user }: { term: string, user: User, languageName: string }): Promise<void | undefined> {
```

Comment:
Should this maybe just be `language` ?

Reply:
Imo, yes, to keep it consistent with the .net client

Reply:
👍